mslib/mscolab/_tests/test_seed.py

```diff
@@ -24,58 +24,61 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 """
-
-import sys
-from mslib.msui.mscolab import MSSMscolabWindow
-from PyQt5 import QtTest, QtWidgets
-from mslib.mscolab.models import User
+from flask import Flask
+from mslib.mscolab.models import User, Project, db
+from mslib.mscolab.file_manager import FileManager
 from mslib.mscolab.conf import mscolab_settings
-from mslib._tests.utils import (mscolab_register_and_login,
-                                mscolab_create_project, mscolab_delete_all_projects,
-                                mscolab_delete_user, mscolab_start_server)
-from mslib.mscolab.seed import add_all_users_default_project, add_user, delete_user
-
-
-PORTS = list(range(19571, 19590))
+from mslib.mscolab.seed import add_user, add_project, add_user_to_project,\
+    delete_user, delete_project, add_all_users_default_project
+from mslib.mscolab.mscolab import handle_db_seed
 
 
 class Test_Seed():
     def setup(self):
-        mscolab_settings.enable_basic_http_authentication = False
-        self.process, self.url, self.app, _, self.cm, self.fm = mscolab_start_server(PORTS, mscolab_settings)
-        QtTest.QTest.qWait(100)
-        self.application = QtWidgets.QApplication(sys.argv)
-        self.window = MSSMscolabWindow(data_dir=mscolab_settings.MSCOLAB_DATA_DIR,
-                                       mscolab_server_url=self.url)
-        with self.app.app_context():
-            response = mscolab_register_and_login(self.app, self.url, 'UV0@uv0', 'UV0', 'uv0')
-            assert response.status == '200 OK'
-            data, response = mscolab_create_project(self.app, self.url, response,
-                                                    path='XYZ', description='Template')
-            assert response.status == '200 OK'
-            data = response.get_data(as_text=True)
-            assert data == 'True'
-            mscolab_register_and_login(self.app, self.url, 'UV1@uv1', 'UV1', 'UV1')
+        handle_db_seed()
+        self.app = Flask(__name__, static_url_path='')
+        self.app.config['SQLALCHEMY_DATABASE_URI'] = mscolab_settings.SQLALCHEMY_DB_URI
+        self.app.config['MSCOLAB_DATA_DIR'] = mscolab_settings.MSCOLAB_DATA_DIR
+        self.app.config['UPLOAD_FOLDER'] = mscolab_settings.UPLOAD_FOLDER
+        self.app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+        db.init_app(self.app)
+
+        self.fm = FileManager(self.app.config["MSCOLAB_DATA_DIR"])
+        self.room_name = "XYZ"
+        self.description = "Template"
+        self.userdata = 'UV0@uv0', 'UV0', 'uv0'
```

Comment:
I would add all dummy user-parameters here. self.user1_data, self.user2_data,.... and reference these variables for adding/deleting users.